library/cohesity_restore_file.py

```diff
@@ -172,7 +177,7 @@
     "changed": true,
     "failed": false,
     "filenames": [
-        "C:\\data\\files"
+        "C:/data/files"
```

Comment:
The windows paths look unconventional.

Reply:
I will fix that, will change it to C:\data\files